python/lsst/ip/isr/isrTask.py

```diff
@@ -46,6 +46,7 @@
 from .defects import Defects
 
 from .assembleCcdTask import AssembleCcdTask
+from .binExposureTask import BinExposureTask
```

Comment:
Can I have you make the same changes to `isrTaskLSST.py` as well?  It should be doing the same binning operation here: https://github.com/lsst/ip_isr/blob/main/python/lsst/ip/isr/isrTaskLSST.py#L1851